src/ocean/util/app/ext/LogExt.d

```diff
@@ -249,3 +248,17 @@ class LogExt : IConfigExtExtension
         return LogUtil.newLayout(name);
     }
 }
+
+
+/// Hack to allow to call a deprecated function without deprecations
+private extern extern(C) void configureOldLoggersLogExtHack (
+    ClassIterator!(LogUtil.Config, ConfigParser) config,
+    LogUtil.MetaConfig m_config,
+    AppenderExternD file_appender, MakeLayoutExternD makeLayout,
+    bool use_insert_appender = false);
+/// Hack because DMD1 applies extern(C) to the delegate type as well...
+private alias extern(D) Appender delegate (istring, Appender.Layout)
+    AppenderExternD;
+/// Ditto
+private alias extern(D) Appender.Layout delegate (cstring)
+    MakeLayoutExternD;
```

Comment:
Interesting idea but I am not sure about its merit. It is not possible to enable it in library code without also enabling it in app code and latter opens up abuse possibility that kind of kills `deprecated` guarantees.

Your hack is pretty cool in that regard actually - it is convoluted enough to only be used if really commited/desperate but at the same time completely solves the problem from the practical PoV.